build: make libavdevice configure check accept libav-12

libav-12 is supported, but configure script expects
a higher libavdevice version than libav-12 provides.

Gentoo-Bug: 603122

diff --git a/wscript b/wscript
index 94db7e296..7f61f54cc 100644
--- a/wscript
+++ b/wscript
@@ -444,7 +444,7 @@ FFmpeg/Libav libraries. You need at least {0}. Aborting.".format(libav_versions_
     }, {
         'name': '--libavdevice',
         'desc': 'libavdevice',
-        'func': check_pkg_config('libavdevice', '>= 57.0.0'),
+        'func': check_pkg_config('libavdevice', '>= 56.1.0'),
     }, {
         'name': 'avcodec-chroma-pos-api',
         'desc': 'libavcodec avcodec_enum_to_chroma_pos API',
